samples/java-eventsourced-shopping-cart/src/main/java/com/example/shoppingcart/view/ShoppingCartViewServiceImpl.java

```diff
@@ -0,0 +1,35 @@
+package com.example.shoppingcart.view;
+
+import com.akkaserverless.javasdk.view.ViewContext;
+import com.example.shoppingcart.domain.ShoppingCartDomain;
+
+// This class was initially generated based on the .proto definition by Akka Serverless tooling.
+//
+// As long as this file exists it will not be overwritten: you can maintain it yourself,
+// or delete it so it is regenerated as needed.
+
+public class ShoppingCartViewServiceImpl extends AbstractShoppingCartViewServiceView {
```

Comment:
If there is nothing but the generated stub then maybe don't check it in and let it regenerate on build?

Reply:
yep, good point